stable/mission-control/values.yaml

```diff
@@ -488,6 +512,32 @@ missionControl:
   internalPort: 8080
   externalPort: 80
 
+  livenessProbe:
+    enabled: true
+    config: |
+      exec:
+        command:
+        - curl 
+        - http://127.0.0.1:{{ .Values.missionControl.internalPort }}/api/v1/system/ping
```

Comment:
Why use curl? There's no guarantee that you would always have curl on the docker image. There's a tendency in JFrog to remove curl from the images every now and then ;) 
Why not use httpGet?

Reply:
httpGet was showing connection refused errors , since the microservices only accept connection from localhost. That is why we had to use curl

Reply:
Strange... the probe is supposed to run in the context of the localhost. This might be an application bug.

Reply:
This looks pretty weird from a customer's perspective...

Reply:
As discussed, I have created a separate ticket to see if it's possible to use httpGet